src/Semantic/Git.hs

```diff
@@ -15,14 +17,21 @@ module Semantic.Git
   , parseEntry
   ) where
 
-import Control.Monad.IO.Class
-import Data.Attoparsec.Text   (Parser)
-import Data.Attoparsec.Text   as AP
-import Data.Char
-import Data.Either            (fromRight)
-import Data.Text              as Text
-import Shelly                 hiding (FilePath)
-import System.IO              (hSetBinaryMode)
+import Prologue
+
+import           Control.Monad.IO.Class
+import           Data.Attoparsec.ByteString.Char8 (Parser)
+import           Data.Attoparsec.ByteString.Char8 as AP
+import qualified Data.Attoparsec.ByteString.Streaming as AP.Stream
+import qualified Data.ByteString.Char8 as BC
+import qualified Data.ByteString.Streaming.Char8 as ByteStream
+import           Data.Char
+import           Data.Either (fromRight)
+import           Data.Text as Text
+import qualified Data.Text.Encoding as Text
+import           Shelly hiding (FilePath)
```

Comment:
What would be required to eliminate this dependency?

Reply:
Reworking `catFile` and `clone` to use a direct process invocation rather than `shelly`. Not onerous, just work I wanted to do in separate PRs to minimize the risk of breakage.